packages/dx-react-grid/src/plugins/table-fixed-columns.test.tsx

```diff
@@ -102,6 +102,7 @@ describe('TableFixedColumns', () => {
     tableColumnsWithFixed.mockImplementation(() => [...columns, {}]);
     tableHeaderColumnChainsWithFixed.mockImplementation(() => [
       [
+        // tslint:disable-next-line: object-shorthand-properties-first
```

Comment:
Cant we remove the `// tslint:...` comment and put `columns` to the first place like we do in the next change?

Reply:
It will affect readability - now 'start' is aligned for both elements

Reply:
OK